composer.json

```diff
@@ -26,7 +26,7 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.2 || ^8.0",
```

Comment:
Requires PHP 8 to be passing CI first - please add that to the patch.

Otherwise, use `--ignore-platform-reqs` for installation for now.